build/docker/zap_common.py

```diff
@@ -107,19 +107,19 @@ def print_rules(alert_dict, level, config_dict, config_msg, min_level, levels, i
 
 
 def inc_ignore_rules(config_dict, key, inc_extra):
-    return config_dict.has_key(key) and config_dict[key] == 'IGNORE'
+    return (key in config_dict) and config_dict[key] == 'IGNORE'
 
 
 def inc_info_rules(config_dict, key, inc_extra):
-    return (config_dict.has_key(key) and config_dict[key] == 'INFO') or (not config_dict.has_key(key) and inc_extra)
+    return (key in config_dict) and config_dict[key] == 'INFO') or ((key not in config_dict) and inc_extra)
```

Comment:
It's missing one opening parenthesis in the left expression, or remove the closing parenthesis before "and"?